eth_worker/eth_src/sql_persistence/interface.py

```diff
@@ -540,12 +540,10 @@ def set_block_range_status(self, start, end, status, filter_id):
         self.session.commit()
 
 
-    def __init__(self, red, session, first_block_hash, PENDING_TRANSACTION_EXPIRY_SECONDS=30):
```

Comment:
wondering on the reasoning against passing this in to the object still? Like we do for the gas price in the processor:

```
processor = EthTransactionProcessor(
    ethereum_chain_id=config.ETH_CHAIN_ID,
    gas_price_wei=w3.toWei(config.ETH_GAS_PRICE, 'gwei'),
```